libs/ui/src/lib/external-link/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Link, type LinkProps } from '@digdir/designsystemet-react';
+// import { ExternalLinkIcon } from '@navikt/aksel-icons';
+
+const ExternalLink = ({ children, ...props }: LinkProps) => (
+    <Link {...props}>
+        {children}
+        {/*<ExternalLinkIcon
+            aria-hidden
+            fontSize='1em'
+            style={{marginLeft:'0.125em'}}
+        />*/}
```

Comment:
og dette?

Reply:
Har kanskje litt lyst til å beholde disse kommentarene inntil videre for å understreke hva denne komponenten "gjorde" - mens vi tester hvordan det funker uten ikonet på alle eksterne lenker.

Reply:
Kan jo eventuelt ha input `showIcon` som er default false?

Reply:
Det er et godt poeng, kan fikse det 👍